datasetinsights/io/tracker/mlflow.py

```diff
@@ -45,31 +45,40 @@ class MLFlowTracker:
     EXP_NAME = "experiment"
     RUN_NAME = "run"
     DEFAULT_RUN_NAME = "run-" + TIMESTAMP_SUFFIX
+    TRACKER = "tracker"
+    MLFLOW_TRACKER = "mlflow"
+    DEFAULT_EXP_NAME = "datasetinsights"
 
-    def __init__(self, mlflow_config):
+    def __init__(
+        self,
+        *,
+        client_id=None,
+        host=None,
```

Comment:
Why not call this `host_id` to make it consistent across the codebase?

Reply:
Same as above!